e2e/scripts/first-run.sh

```diff
@@ -9,3 +9,7 @@ hostnamectl set-hostname "$hostname"
 
 echo "Adding hostname to hosts file..."
 echo "127.0.0.1 $hostname" >> /etc/hosts
+
+echo "Removing ssh configuration overrides..."
```

Comment:
hum? is it related to that PR? I think the echo isn’t descriptive enough why we are doing this.